easybuild/easyconfigs/p/ParaView/ParaView-5.8.0-foss-2020a-Python-3.8.2-mpi.eb

```diff
@@ -41,7 +41,7 @@ configopts += '-DPARAVIEW_ENABLE_FFMPEG=ON '
 
 # OpenGL & Mesa
 configopts += '-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s ' % SHLIB_EXT
-configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include'
+configopts += '-DOPENGL_INCLUDE_DIR=$EBROOTMESA/include '
 configopts += '-DVTK_OPENGL_HAS_OSMESA=ON '
```

Comment:
```suggestion
```

Reply:
Doesn't this imply dropping the `Mesa` dep too?